### DISCUSSION TITLE
Xaml Islands

### DISCUSSION THREAD

**Discussion by Apskalle** [microsoft/WindowsAppSDK#1696] [Q&A] [answered]
When in time do you estimate supporting xaml islands in .net5, including community toolkit controls?

Cheers,

Martin

**Comment by btueffers** [comment, marked as answer]
Hi @Apskalle, we don't have further updates at this time. As the [WinUI3 Roadmap](https://github.com/microsoft/microsoft-ui-xaml/blob/main/docs/roadmap.md#legend) states, it is still planned for future update.
